protected/models/Csv.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: fivium-isaac
+ * Date: 12/1/17
+ * Time: 2:49 PM
+ */
+
+class Csv
+{
+    public $csvFile;
+
+    public $dataTable;
+
+    public function hasErrors()
+    {
+
```

Comment:
?

Reply:
the form doesn't like not having this function